pyvirtualcam/native_macos/main.mm

```diff
@@ -0,0 +1,165 @@
+// Rest of files is stolen from:
+// - https://github.com/obsproject/obs-studio/tree/516ed4458dbc74b27b425533789d4e9dd95b923b/plugins/mac-virtualcam
+// - https://github.com/descampsa/yuv2rgb/tree/c51a543cab4322598047bcd86f5d5fd680e9e6f2
+// Consider migration to https://github.com/johnboiles/coremediaio-dal-minimal-example ?
+
+#include <stdexcept>
+#include <pybind11/pybind11.h>
+#include <pybind11/numpy.h>
+#include "OBSDALMachServer.h"
+#include "Defines.h"
+#include <mach/mach_time.h>
+#include <cstdint>
+
+namespace py = pybind11;
+
+#define UNUSED_PARAMETER(arg)
```

Comment:
This is a custom macro so we can continue to use the original code from the macOS obs plugin

Reply:
I don't think this is used anywhere (anymore?), probably can be removed.

Reply:
Oh right, I probably cleaned this up a bit.

Originally main.mm was almost 1:1 copy of the plugin-main.mm from the OBS plugin. I later changed it to be closer to the Windows main.cpp (and also restructured it a bit, probably removing use of this macro).

Reply:
Removed